src/PE/signature/SignerInfo.cpp

```diff
@@ -67,21 +67,25 @@ std::ostream& operator<<(std::ostream& os, const SignerInfo& signer_info) {
 
   constexpr uint8_t wsize = 30;
   const issuer_t& issuer = signer_info.issuer();
+  std::string issuer_str = std::get<0>(issuer);
 
-  std::string issuer_str = std::accumulate(
-      std::begin(std::get<0>(issuer)),
-      std::end(std::get<0>(issuer)),
+  const std::vector<uint8_t>& sn = std::get<1>(issuer);;
+  std::string sn_str = std::accumulate(
+      std::begin(sn),
+      std::end(sn),
       std::string(""),
-      [] (std::string lhs, const std::pair<std::string, std::string>& p) {
-        std::string s = oid_to_string(std::get<0>(p)) + std::string("=") + std::get<1>(p);
-        return lhs.empty() ? s : lhs + " " + s;
+      [] (std::string lhs, uint8_t x) {
+        std::stringstream ss;
```

Comment:
I think you are missing an ``#include <sstream>`` at the beginning.

Reply:
weird, it built fine for me without it... I added the include, though, so hopefully that addresses the issue.  Thanks!